content/blog/2023-04-01-key-expressions-1.md

```diff
@@ -20,14 +20,31 @@ Early in Zenoh's design, we found that the ability to act not just on a single k
 
 To do so, a "wildcard" syntax similar to those of glob patterns was introduced, and the Key Expression Language (KEL) was born.
 
-## Specifying the KEL.
+![key-expression](../../img/admin-tool/zenoh-keyexp.png)
 
-But there still was a major caveat: the language was largely underspecified, leaving the behaviour of certain patterns up to interpretation. To remedy this, we went the way most languages tend to, and stopped considering any text string as a valid KE, by defining a proper language for KEs.
+# Specifying the KEL.
+
+But there still was a major caveat: the language was largely _underspecified_, leaving the behaviour of certain patterns up to interpretation. To remedy this, we went the way most languages tend to, and stopped considering any text string as a valid KE, by defining a proper language for KEs.
 
 This one done in a classic 3 steps program: make KEs better, ???, profit. Want the actual steps?
 1. Redefine KEs as a `/`-separated list of non-empty UTF-8 strings called "chunks". With this, the ambiguities (and many internal debates) about the expected behaviours of `a/b/` and `a//b` in relation to `a/b` were finally laid to rest, since only the latter was a valid KE.
+
+![key-expression-chunks](../../img/admin-tool/keyexp-chunks.png)
```

Comment:
I'm not sure the pictures add anything to the blog: most Zenoh users aren't familiar with the tool yet (and may very well never have to use it).

If it were me, I'd get confused trying to find the purpose of the picture and where it may have come from.